pkg/utils/environment_variable.go

```diff
@@ -0,0 +1,64 @@
+package utils
+
+import (
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/log"
+)
+
+var defaultEnvVarValueOrDefault = envVarValueOrDefault
```

Comment:
what is happening on this line?

Reply:
we set the function as a variable so it's clearer to the reader that the function being used in the one defined at the env of the file